src/libs/lambda.ts

```diff
@@ -3,10 +3,5 @@ import middyCors from "@middy/http-cors";
 import middyJsonBodyParser from "@middy/http-json-body-parser";
 import middyErrorHandler from "@middy/http-error-handler";
 
-export const middyfy = (handler) => {
-  return middy(handler).use([
-    middyJsonBodyParser(),
-    middyErrorHandler(),
-    middyCors(),
-  ]);
-};
+export const middyfy = (handler: any): any =>
+  middy(handler).use([middyJsonBodyParser(), middyErrorHandler(), middyCors()]);
```

Comment:
Maybe you can do something like this for the type so that don't have to use `any`?

```suggestion
import { Context, Handler } from 'aws-lambda';

export const middyfy = <TEvent, TContext = Context>(
  handler: Handler<TEvent, TContext>
): MiddyfiedHandler<TEvent, TContext> => {
  return middy(handler).use([
    middyJsonBodyParser(),
    middyErrorHandler(),
    middyCors(),
  ]);
};
```

Reply:
This is great, thanks!